Source/ResourceLibUnitTests/ResourceTests.cs

```diff
@@ -5,6 +5,7 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Linq;

```

Comment:
Sorry, this is an artifact from an earlier implementation of mine. I'll delete it.